Ccount.c

```diff
@@ -0,0 +1,22 @@
+#include <stdio.h>
```

Comment:
Perfect job! Just, again, please include comments! This is very important, especially if you ever submit code that doesn't compile or run or work correctly.